tutorials/operators/02_gradients_framework.ipynb

```diff
@@ -215,19 +215,11 @@
      "slide_type": "subslide"
     }
    },
-   "outputs": [
-    {
-     "name": "stdout",
-     "output_type": "stream",
-     "text": [
-      "Gradient [(1.414213562373095+1.12e-16j), 0j]\n"
-     ]
-    }
-   ],
+   "outputs": [],
    "source": [
     "# Assign the parameters and evaluate the gradient\n",
-    "grad_result = grad.assign_parameters(value_dict).eval()\n",
-    "print('Gradient', grad_result)"
+    "# grad_result = grad.assign_parameters(value_dict).eval()\n",
```

Comment:
In talking to @Cryoris he suggested replacing `~StateFn` with `StateFn(..., is_measurement=True)` to avoid the failure this is hitting. If we do that we should be able to uncomment this line

Reply:
I made the change and it fixed the problem.